src/io/GeantGeometryMap.cc

```diff
@@ -0,0 +1,109 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2020 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file GeantGeometryMap.cc
+//---------------------------------------------------------------------------//
+#include "GeantGeometryMap.hh"
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Construct/destruct with defaults
+ */
+GeantGeometryMap::GeantGeometryMap()  = default;
+GeantGeometryMap::~GeantGeometryMap() = default;
+
+//---------------------------------------------------------------------------//
+/*!
+ * Return the mat_id for a given vol_id
+ */
+GeantGeometryMap::mat_id GeantGeometryMap::get_matid(vol_id& volume_id)
+{
+    auto iter = volid_to_matid_.find(volume_id);
+    REQUIRE(iter != volid_to_matid_.end());
+    return iter->second;
+}
+
+//---------------------------------------------------------------------------//
+/*!
+ * Return the GeantVolume associated with the vol_id
+ */
+GeantVolume GeantGeometryMap::get_volume(vol_id& volume_id)
```

Comment:
These should be `const vol_id&` (here and elsewhere)

Reply:
Also since these structs are large, return const references instead of values (needless copy).